### ISSUE TITLE
Search and other forms of navigation

### ISSUE THREAD

**Issue by Angelsaur** [alan-turing-institute/AutSPACEs#65]
### Summary

You’ll need to do some checks to see how accessible your website’s navigation is. Navigation includes things like search, site maps and navigation categories

### Deadline 

20/02/2021

### What needs to be done?

**Check your navigation behaves consistently**
Things like search bars and breadcrumb trails (the list of links that appear at the top of the page to show where in your website’s navigation you currently are) should behave in the same way wherever they appear on your website.

You can check for this by going to a few of your sample pages and checking whether the navigational elements are presented consistently. If things like search boxes or navigation menus appear in different places across the pages, you’re probably not meeting this requirement.

**Check links do not trigger on the down-press of a mouse**
To open a link, a user has to click their mouse (known as a ‘down event’) and release it (known as an ‘up event’).

You need to check that this is how your links behave. Check a sample of your content pages and interact with some of the links - specifically to see whether the links are opened on the down click of the mouse, or whether they only open once you release the mouse button.

If they’re triggered on the down event, you’re likely not meeting this requirement.
